website/src/pages/ga4gh/drs/v1/objects/[id]/index.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+import crypto from 'crypto';
+
+import type { APIRoute } from 'astro';
+import { err, ok, Result } from 'neverthrow';
+
+import { getConfiguredOrganisms } from '../../../../../../config';
+import { LapisClient } from '../../../../../../services/lapisClient';
+import type { ProblemDetail } from '../../../../../../types/backend';
+import { parseAccessionVersionFromString } from '../../../../../../utils/extractAccessionVersion';
+
+/**
+ * DRS Object type definition according to GA4GH DRS v1.x
+ * Snake case is required by the GA4GH DRS specification
+ *
+ * @see https://ga4gh.github.io/data-repository-service-schemas/docs/
+ */
+
+interface DrsObject {
+    id: string;
+    name: string;
+
+    self_uri: string;
+    size: number;
+
+    created_time: string;
+
+    updated_time: string;
+    version: string;
+
+    mime_type: string;
+    checksums: {
+        type: string;
+        checksum: string;
+    }[];
+
+    access_methods: {
+        type: string;
+
+        access_id: string;
+
+        access_url: string | null;
+        region: string | null;
+        headers: Record<string, string> | null;
```

Comment:
In the spec I do not see headers instead I see:
https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.5.0/docs/#tag/Objects/operation/GetObject

```suggestion
        "cloud": "aws, gcp, or azure",
      "region": "us-east-1",
      "available": true,
      "authorizations": {
        "drs_object_id": "string",
        "supported_types": [
          "None"
        ],
        "passport_auth_issuers": [
          "string"
        ],
        "bearer_auth_issuers": [
          "string"
        ]
      }
```

Reply:
Thanks for pointing out @theosanderson that these are not required fields; https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.5.0/docs/#tag/AccessMethodModel (however we can probably still remove headers)